doc-style/action.yml

```diff
@@ -71,6 +71,37 @@ runs:
       uses: actions/checkout@v4
       if: inputs.checkout == 'true'
 
+    - uses: ansys/actions/_logging@main
+      with:
+        level: "INFO"
+        message: >
+          Ensure Vale configuration follows the right layout for the specified
+          version of Vale.
+
+    - name: "Check Vale's version"
+      shell: bash
+      run: |
+        version=${{ inputs.vale-version }}
+        vale_major_version=$( echo $version | cut -d '.' -f 1)
+        echo "VALE_MAJOR_VERSION=$(echo $major)" >> $GITHUB_ENV
+
+    - name: "Verify Vale's configuration layout according to selected version"
+      shell: bash
+      run: |
+        if [ $VALE_MAJOR_VERSION -ge 3 ]; then
+          if [ ! -d ./doc/styles/config/vocabularies/ANSYS ]; then
```

Comment:
I would just check up to ``vocabularies`` - maybe some other users out there want to benefit from the action?